Makefile.inc

```diff
@@ -31,73 +31,58 @@ include $(SHIPYARD_DIR)/Makefile.versions
 # Process extra flags from the `using=a,b,c` optional flag
 
 ifneq (,$(filter libreswan,$(_using)))
-override DEPLOY_ARGS += --cable_driver libreswan
+export CABLE_DRIVER = libreswan
```

Comment:
Since these are exported further down, how about

    CABLE_DRIVER = lighthouse

Same for the other variables.